### ISSUE TITLE
Error 503 on start

### ISSUE THREAD

**Issue by abea** [kalabox/kalabox#785]
```
Error 503 Backend fetch failed

Backend fetch failed

Guru Meditation:

XID: 21

Varnish cache server
```

All of a sudden today I got this error on `kbox start`. Usually errors like this are resolved by rebuilding or restarting, but nothing seems to be working here. The whole system was working fine yesterday evening, though I don't think anything has really changed since then.

**Comment by labboy0276** [comment]
Hello!

Please see this issue: https://github.com/kalabox/kalabox/issues/748

However quick and easy is to add this to your settings.php:

```
if (PANTHEON_ENVIRONMENT === 'kalabox') {
 setcookie("NO_CACHE", 'kalabox');
}
```

**Comment by abea** [comment]
Thanks, John. I added that to setting.local.php and rebuilt/restarted with the same result.

**Comment by pirog** [comment]
@abea so i think the issue here is related to varnish not correctly grabbing the backend IP information for the appserver although restarting generally seems to resolve that. 

Were you able to eventually get this to work or did you just rebuild the app?

**Comment by dustinleblanc** [comment]
@pirog I think @abea is temporarily using MAMP while things are sorted out. I've jumped back to Kbox for my Pantheon apps but I'm keeping `drush rs` in my back pocket for emergencies :)

Alex if you run across downtime it'd be cool to try rebuilding and see if resolves itself.

**Comment by dustinleblanc** [comment]
FYI, this happened to me this morning and a restart fixed it

**Comment by jdleonard** [comment]
I'm also encountering this (again) when visiting my kalabox site after a `kbox start` immediately following an OS restart. Possibly related: #818, #749.

`kbox restart` didn't help.

`kbox rebuild` resulted in 500s instead of 503s.

'kbox pull' resolved the issue (while overwriting local changes).

**Comment by Mchampsee** [comment]
Getting something similar bringing WordPress down from Pantheon. I tried rebuilding, pulling as well as restarting the service and rebooting the machine to no avail.

**Comment by ptmkenny** [comment]
I'm having the same issue.  I just set up kbox today.  I'm using kbox-macosx-amd64-v0.10.6-alpha6.  I have two Drupal 7 sites, both of which were set up fine.

However, I have one Drupal 8 site that is vanilla Drupal 8, and all I've done on Pantheon is log in and create an admin user, so it is basically everything at the default values.  When I set up the site, it worked for a few minutes, but after that I have since been consistently getting Error 503.  I tried restart and pull several times, and I destroyed the site and started over, but I haven't been able to get it working since.

**Comment by labboy0276** [comment]
Sometimes this happens with me.  The easiest way is to do either a kbox
restart or a kbox rebuild and kbox start.

The next version of kalabox is going to be out soonish and you will no
longer have these issues.

On Wed, Jan 13, 2016 at 5:30 AM, ptmkenny <notifications@github.com> wrote:

> I'm having the same issue. I just set up kbox today. I'm using
> kbox-macosx-amd64-v0.10.6-alpha6. I have two Drupal 7 sites, both of which
> were set up fine.
>
> However, I have one Drupal 8 site that is vanilla Drupal 8, and all I've
> done on Pantheon is log in and create an admin user, so it is basically
> everything at the default values. When I set up the site, it worked for a
> few minutes, but after that I have since been consistently getting Error
> 503. I tried restart and pull several times, and I destroyed the site and
> started over, but I haven't been able to get it working since.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kalabox/kalabox/issues/785#issuecomment-171248523>.
>



-- 
*John Ouellet*
*Sr Developer & Support Manager*
415-470-3076
www.kalamuna.com

Stay connected with Kalamuna on: Twitter
<http://t.sidekickopen01.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1pwYhxW8qlTmx64kckdW8pTc_456dT6hf3BRzQ602?t=http%3A%2F%2Ftwitter.com%2Fkalamuna&si=5428072445640704&pi=f16b515d-1f50-46c7-aebf-11a0c79916b9>
, Linkedin
<http://t.sidekickopen01.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1pwYhxW8qlTmx64kckdW8pTc_456dT6hf3BRzQ602?t=http%3A%2F%2Fwww.linkedin.com%2Fcompany%2F2584606&si=5428072445640704&pi=f16b515d-1f50-46c7-aebf-11a0c79916b9>,
and Facebook
<http://t.sidekickopen01.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1pwYhxW8qlTmx64kckdW8pTc_456dT6hf3BRzQ602?t=http%3A%2F%2Fwww.facebook.com%2Fkalamunatech&si=5428072445640704&pi=f16b515d-1f50-46c7-aebf-11a0c79916b9>

**Comment by johnnypea** [comment]
I tried everything mentioned here. I can't get the environment running. Anybody found a working solution?

OS X 10.11.2

**Comment by jdleonard** [comment]
I bailed for now; using MAMP, but I much prefer Kalabox!

**Comment by labboy0276** [comment]
When I see this happening (note it is fixed, so I hear in 0.11.0 which will be out soonish) after a kbox start

I make sure this in settings.php
```
if (PANTHEON_ENVIRONMENT === 'kalabox') {
 setcookie("NO_CACHE", 'kalabox');
}
```
Then I just do a kbox restart and all is well.

Once, it just wouldn't comply and I did a complete uninstall and reinstall and it was happy again.

**Comment by pirog** [comment]
So this should be fixed in the v0.11.0 release we are doing on Friday. Beyond the backend 503 being resolved you now can directly bypass varnish by going to "appserver.appname.kbox".
